README.ios.md

```diff
@@ -0,0 +1,43 @@
+# Glean SDK
+
+## Documentation
+
+All documentation is available online:
+
+## [The Glean SDK Book][book]
+
+## Overview
+
+Refer to the documentation for [using and developing the Glean SDK][book].
+
+For an overview of Glean beyond just the SDK, see the [section in the Firefox data docs](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
+
+**Note: The Glean SDK requires at least [Rust 1.34.2](https://blog.rust-lang.org/2019/05/14/Rust-1.34.2.html). Older versions are untested.**
+
+## Contact
```

Comment:
Instead of repeating this, can we link to the relevant section of the [main readme file](https://github.com/mozilla/glean/blob/master/README.md)?

Reply:
This readme is rendered as the main page for the swift-specific API documentation.
I'm considering replacing it completely with a big link to the main page instead.